packages/backend/src/server.ts

```diff
@@ -16,7 +16,35 @@ import rateLimiter from './common/middleware/rateLimiter';
 import requestLogger from './common/middleware/requestLogger';
 import { env } from './common/utils/envConfig';
 
-const logger = pino({ name: 'server start' });
+const loggerOptions = [{ level: 'info' }];
```

Comment:
Can we move the logger options to a logger config file  ? it can be in the common/utils folder logConfig.ts and exports the loggerOptions object.